src/thermompnn/model/modules.py

```
@@ -50,30 +60,30 @@ def get_protein_mpnn(cfg, version='v_48_020.pt'):
 
 
 class SideChainPositionalEncodings(nn.Module):
-    def __init__(self, num_embeddings, period_range=[2,1000], max_relative_feature=32, af2_relpos=False):
-        super(SideChainPositionalEncodings, self).__init__()
+    def __init__(self, num_embeddings, period_range=[2, 1000], max_relative_feature=32, af2_relpos=False):
```

Ruff (0.8.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)